.github/workflows/docs-preview.yml

```diff
@@ -2,7 +2,7 @@ name: Docs Preview
 on: [status]
 jobs:
   circleci_artifacts_redirector_job:
-    if: github.event.context == 'ci/circleci: Build Docs Preview'
+    if: "github.event.context == 'ci/circleci: Build Docs Preview'"
```

Comment:
I haven't seen this syntax before. Usually the syntax is
```
if: ${{ github.event.context == ... }}
```

Reply:
I think maybe the `if` key evaluates itself as an expression automatically. The braces aren't there in the examples https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#status

Reply:
So what does wrapping the expression in quotes do?

Reply:
Makes it valid YAML syntax. YAML usually parses things as a string unless it looks like something else. In this case, I think the `:` confuses it.

Reply:
Okay, that makes sense. It's supposed to a YAML string but then GA will evaluate that string as an expression.